doc/system_requirements.rst

```diff
@@ -0,0 +1,33 @@
+System and Tool Requirements
+============================
+
+The Ibex CPU core is written in SystemVerilog.
+We try to achieve a balance between the used language features (as described in our `style guide <https://github.com/lowRISC/style-guides/blob/master/VerilogCodingStyle.md>`_) and reasonably wide tool support.
+
+We regularly use and test the following tools for working with the RTL code of Ibex.
```

Comment:
Is this statement perhaps a bit strong? We've heard positive success stories from the tools listed, but the core Ibex developers don't use all of them in their day-to-day flow, and we don't have automated testing set up for them?

Reply:
Slightly reworded that one, thanks!